src/index.js

```diff
@@ -11,7 +11,12 @@ import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import Glean from '@mozilla/glean/web';
 
+// Initialize Glean. ToDo: Replace uploadEnabled: true with a user preference
+const APP_NAME = "debug-ping-view"
+Glean.setDebugViewTag(APP_NAME);
```

Comment:
```suggestion
```
Let's not land automatic submission to the debug ping view please (e.g. let's remove this line).

Reply:
Done.